frontend/src/components/clinics/clinics.tsx

```diff
@@ -65,6 +66,9 @@ const Clinics: React.FC = () => {
         )}
         <div className={styles.clinicsWrapper}>
           <div className={styles.clinicsContainer}>
+
+            {!clinics.length && <NoData label={NoDataLabels.NO_CLINICS} />}
+
             {clinics.map((clinic) => (
               <Clinic key={clinic?.id} clinic={clinic} />
             ))}
```

Comment:
```suggestion
  hasClinics ? clinics.map((clinic) => <Clinic />) : <NoDataPlaceholder />
```
Change for all in this way